docs/contributing.md

```diff
@@ -0,0 +1,132 @@
+# Contributing
+
+This document describes how to contribute to the development of this software.
+
+## Contribution Workflow
+
+### Create an Issue
```

Comment:
```suggestion
### Create an Issue or Feature Request
```

Reply:
A "Feature Request" is a subset of an Issue, they all reside under "Issues" whether they have used the "Feature Request" or "Bugs" template.